gitlab-push/handler.go

```diff
@@ -0,0 +1,212 @@
+package function
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/openfaas/openfaas-cloud/sdk"
+)
+
+const Source = "gitlab-push"
+
+// Handle a serverless request
+func Handle(req []byte) string {
+
+	event := os.Getenv("Http_X_Gitlab_Event")
```

Comment:
Can you add a unit test for these scenarios?

Reply:
Yeah